docs/release-notes/8.0.asciidoc

```diff
@@ -1,6 +1,55 @@
 [[release-notes-header-8.0.0]]
 == 8.0
 
+[discrete]
+[[release-notes-8.0.0]]
+=== 8.0.0
+
+[discrete]
+[[post-upgrade-reqs-8.0.0]]
+==== Post upgrade requirements
+* Changes to the indicator match rule's default threat indicator path (<<rule-ui-advanced-params, *Indicator prefix override*>>) might require you to update existing rules, or create new ones, after upgrading to {stack} version 8.0.0. Be mindful of the following ({pull}118821[#118821]):
+** If an indicator match rule's default threat indicator path was not defined before upgrading, it will receive the default value `threatintel.indicator` during the upgrade process. This allows the rule to continue using indicator data ingested by {filebeat} version 7.x. If a custom value was defined before the upgrade, the value remains the same.
```

Comment:
```suggestion
** If an indicator match rule's default threat indicator path was not defined before the upgrade, it will default to `threatintel.indicator` after the upgrade. This allows the rule to continue using indicator data ingested by {filebeat} version 7.x. If a custom value was defined before the upgrade, it will not change.
```